native/packages/react-native-bpk-component-card/stories.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react-native';
+import { StyleSheet, View, Platform } from 'react-native';
+import BpkCard from './index';
+import BpkText from '../react-native-bpk-component-text';
```

Comment:
Better to import this as a module, i.e. add `react-native-bpk-component-text` as a `devDependency` and import it that way. Lerna does some clever symlinking so that it doesn't actually install the dependency, it just references it inside the monorepo.